### ISSUE TITLE
There is no available SQL API with all supported clients & operations 

### ISSUE THREAD

**Issue by katbyte** [Azure/azure-sdk-for-go#6276]
### Bug Report

- What happened?

We have to import upwards of three different SQL api versions to fully support the sql resources:
`github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql`
`github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-10-01-preview/sql`
`github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2018-06-01-preview/sql`

so we get this:
```
import (
	sql201703 "github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql"
	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-10-01-preview/sql"
	sql201806 "github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2018-06-01-preview/sql"
)

type Client struct {
	ElasticPoolsClient *sql.ElasticPoolsClient
	ElasticPoolsClient                                 *sql.ElasticPoolsClient
	DatabaseVulnerabilityAssessmentRuleBaselinesClient *sql201703.DatabaseVulnerabilityAssessmentRuleBaselinesClient
	ServerSecurityAlertPoliciesClient                  *sql201703.ServerSecurityAlertPoliciesClient
	ServerVulnerabilityAssessmentsClient               *sql201806.ServerVulnerabilityAssessmentsClient
}
```

and it becomes awfully confusing.

- What did you expect or want to happen?
one unified API containing all clients and operations

- How can we reproduce it?
try to provision all sql resources in a project

- Anything we should know about your environment.
nope

**Comment by ArcturusZhang** [comment]
Yes, this is truly a problem now. However, they currently have a tag `package-composite-v3` in the sql service [readme](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md), which is a combination of all existing api-version of sql. I could submit a release for this tag called `v3`. Similar situation happens to the security service ([readme](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/security/resource-manager/readme.go.md)), they have an api-version v3 to cover all possible api-versions.

**Comment by ArcturusZhang** [comment]
Hi @katbyte I released package of sql v3.0:
`github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/v3.0/sql`
